lib/vtt.js

```diff
@@ -1292,7 +1301,7 @@
         // Create the region, using default values for any values that were not
         // specified.
         if (settings.has("id")) {
-          var region = new self.window.VTTRegion();
+          var region = new (self.vttjs.VTTRegion || self.window.VTTRegion)();
```

Comment:
Let's move this and the `VTTCue` part to properties on the `WebVTTParser` object, so we just `new` that every time instead of having to check here.